src/languages/stan.js

```diff
@@ -53,421 +74,241 @@ export default function(hljs) {
-    'frechet_cdf',
-    'frechet_lccdf',
-    'frechet_lcdf',
-    'frechet_lpdf',
-    'frechet_rng',
-    'gamma_cdf',
-    'gamma_lccdf',
-    'gamma_lcdf',
-    'gamma_lpdf',
-    'gamma_p',
-    'gamma_q',
-    'gamma_rng',
-    'gaussian_dlm_obs_lpdf',
-    'get_lp',
-    'gumbel_cdf',
-    'gumbel_lccdf',
-    'gumbel_lcdf',
-    'gumbel_lpdf',
-    'gumbel_rng',
-    'head',
-    'hypergeometric_lpmf',
-    'hypergeometric_rng',
-    'hypot',
-    'inc_beta',
-    'int_step',
-    'integrate_ode',
-    'integrate_ode_bdf',
-    'integrate_ode_rk45',
-    'inv',
-    'inv_Phi',
-    'inv_chi_square_cdf',
-    'inv_chi_square_lccdf',
-    'inv_chi_square_lcdf',
-    'inv_chi_square_lpdf',
-    'inv_chi_square_rng',
-    'inv_cloglog',
-    'inv_gamma_cdf',
-    'inv_gamma_lccdf',
-    'inv_gamma_lcdf',
-    'inv_gamma_lpdf',
-    'inv_gamma_rng',
-    'inv_logit',
-    'inv_sqrt',
-    'inv_square',
-    'inv_wishart_lpdf',
-    'inv_wishart_rng',
-    'inverse',
-    'inverse_spd',
-    'is_inf',
-    'is_nan',
-    'lbeta',
-    'lchoose',
-    'lgamma',
-    'lkj_corr_cholesky_lpdf',
-    'lkj_corr_cholesky_rng',
-    'lkj_corr_lpdf',
-    'lkj_corr_rng',
-    'lmgamma',
-    'lmultiply',
-    'log',
-    'log10',
-    'log1m',
-    'log1m_exp',
-    'log1m_inv_logit',
-    'log1p',
-    'log1p_exp',
-    'log2',
-    'log_determinant',
-    'log_diff_exp',
-    'log_falling_factorial',
-    'log_inv_logit',
-    'log_mix',
-    'log_rising_factorial',
-    'log_softmax',
-    'log_sum_exp',
-    'logistic_cdf',
-    'logistic_lccdf',
-    'logistic_lcdf',
-    'logistic_lpdf',
-    'logistic_rng',
-    'logit',
-    'lognormal_cdf',
-    'lognormal_lccdf',
-    'lognormal_lcdf',
-    'lognormal_lpdf',
-    'lognormal_rng',
-    'machine_precision',
-    'matrix_exp',
-    'max',
-    'mdivide_left_spd',
-    'mdivide_left_tri_low',
-    'mdivide_right_spd',
-    'mdivide_right_tri_low',
-    'mean',
-    'min',
-    'modified_bessel_first_kind',
-    'modified_bessel_second_kind',
-    'multi_gp_cholesky_lpdf',
-    'multi_gp_lpdf',
-    'multi_normal_cholesky_lpdf',
-    'multi_normal_cholesky_rng',
-    'multi_normal_lpdf',
-    'multi_normal_prec_lpdf',
-    'multi_normal_rng',
-    'multi_student_t_lpdf',
-    'multi_student_t_rng',
-    'multinomial_lpmf',
-    'multinomial_rng',
-    'multiply_log',
-    'multiply_lower_tri_self_transpose',
-    'neg_binomial_2_cdf',
-    'neg_binomial_2_lccdf',
-    'neg_binomial_2_lcdf',
-    'neg_binomial_2_log_lpmf',
-    'neg_binomial_2_log_rng',
-    'neg_binomial_2_lpmf',
-    'neg_binomial_2_rng',
-    'neg_binomial_cdf',
-    'neg_binomial_lccdf',
-    'neg_binomial_lcdf',
-    'neg_binomial_lpmf',
-    'neg_binomial_rng',
-    'negative_infinity',
-    'normal_cdf',
-    'normal_lccdf',
-    'normal_lcdf',
-    'normal_lpdf',
-    'normal_rng',
-    'not_a_number',
-    'num_elements',
-    'ordered_logistic_lpmf',
-    'ordered_logistic_rng',
+    // Integer-Valued Basic Functions
```

Comment:
Did you get this categorized list ready made from somewhere (or a script)?  If so we should link to it... it's nice to read for sure... but if not, who is going to maintain this over time - and would not a simple alphabetical list be preferable?  "Nice" lists like this often become a mess as no one knows where to add "one-offs" as often happens during real maintenance of a grammar over the long-term.

Reply:
You can get all the signatures this way:

clone the [stan-docs repo](https://github.com/stan-dev/docs) then cd into it run the python script with the major and minor version numbers:
```bash
python3 extract_function_sigs.py 2 28
```
then run a bash command to extract only the unique names
```bash
cut -d ';' -f1 stan-functions-2_28.txt | uniq -u > stan_functions_list-2_28.txt
```
I'll update with the alphabetical list and then I can remove the `_lpdf` stuff as well since this can be updated easily

Reply:
> You can get all the signatures this way:

Lets comment this in the source to aid future maintenance.